packages/slate-editor/src/icons/Component-Button.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <rect x="5.8" y="9.8" width="20.4" height="12.4" stroke="currentColor" stroke-width="1.6"/>
+  <path fill-rule="evenodd" clip-rule="evenodd" d="M9.99995 15.3999C9.99995 15.179 10.179 14.9999 10.4 14.9999L21.8 14.9999C22.0209 14.9999 22.2 15.179 22.2 15.3999L22.2 16.1999C22.2 16.4208 22.0209 16.5999 21.8 16.5999L10.4 16.5999C10.179 16.5999 9.99995 16.4208 9.99995 16.1999L9.99995 15.3999Z" />
+</svg>
```

Comment:
This SVG doesn't look as required.

In this PR:
![image](https://github.com/prezly/slate/assets/370680/a564d0f2-d143-48f3-aa41-782f1dc67eb1)

In the designs:
![image](https://github.com/prezly/slate/assets/370680/75cc92d6-249a-4791-8dd7-ef2941238aa4)

Use this SVG instead:

```suggestion
<svg width="32" height="32" viewBox="0 0 32 32"  xmlns="http://www.w3.org/2000/svg">
  <rect x="5.8" y="9.8" width="20.4" height="12.4" fill="none" stroke="currentColor" stroke-width="1.6" />
  <path fill-rule="evenodd" clip-rule="evenodd" d="M9.99995 15.3999C9.99995 15.179 10.179 14.9999 10.4 14.9999L21.8 14.9999C22.0209 14.9999 22.2 15.179 22.2 15.3999L22.2 16.1999C22.2 16.4208 22.0209 16.5999 21.8 16.5999L10.4 16.5999C10.179 16.5999 9.99995 16.4208 9.99995 16.1999L9.99995 15.3999Z"/>
</svg>
```